papers/Catlett_Amanda/UCET.rst

```diff
@@ -162,11 +161,11 @@ UCET utilizes the following widgets to gather user input:
 
 UCET utilizes indicators.Number, Tabulator, and graphs to visualize the outputs of the coastal engineering models.  A
 single number is shown using indicators.Number and graph data is displayed using the Tabulator widget to show the data
-of the graph. The graphs are created using HoloViews and will have tool options such as pan, zooming, and saving.
+of the graph. The graphs are created using HoloViews and have tool options such as pan, zooming, and saving.
 Buttons are used to calculate, save the current run, and save the graph data.
 
 All of these widgets are organized into 5 panels: title, options, inputs, outputs, and graph. The
-BaseGui/WaterTypeGui/TabularDataGui have methods that will organize the widgets within the 5 panels that most tools
+BaseGui/WaterTypeGui/TabularDataGui have methods that organize the widgets within the 5 panels that most tools
```

Comment:
@acatlett8 Another figure here that illustrates the 5 panels would help clarify this part.